store-api-server/src/payment/service/payment.service.ts

```diff
@@ -868,6 +807,123 @@ WHERE provider IN ('tezpay', 'wert')
     }
   }
 
+  @Cron(CronExpression.EVERY_MINUTE)
+  async checkPendingSimplex() {
+    const pendingPaymentIds = await this.conn.query(
+        `
+SELECT
+  payment_id
+FROM payment
+WHERE provider IN ('simplex')
```

Comment:
Minor: bit nicer to just do `provider = 'simplex'` here instead of using the `IN`

Reply:
Ok, let's fixed it.